LICENSE.bin

```diff
@@ -0,0 +1,349 @@
+                                 Apache License
```

Comment:
@manuzhang we were dinged during or 0.8.1 release when one of the rejecting apache voters said we shouldn't include binary licenses unless we're releasing binary artifacts. So we need to have a LICENSE file for source and a LICENSE file for binary. In this case when I build the binary tar.gz I copy the LICENSE.bin to LICENSE.